app/src/main/java/org/mozilla/tv/firefox/FirefoxApplication.kt

```diff
@@ -78,6 +89,18 @@ open class FirefoxApplication : LocaleAwareApplication() {
         }
     }
 
+    private fun initPush() {
+        var admAvailable = ADM(applicationContext).isSupported
+        // Only install push feature if ADM is available on this device
+        if (admAvailable) {
+            Log.log(tag = DEFAULT_LOGTAG, message = "ADM not available")
+            // This installs the [pushFeature] as a singleton
+            PushProcessor.install(serviceLocator.pushRepo.pushFeature)
+        } else {
+            android.util.Log.i(DEFAULT_LOGTAG, "ADM is not available on this device.")
```

Comment:
If ADM is not available, that means send tab is unavailable on this device. Can you file a follow-up to determine:
- Are there any devices we ship to without ADM?
- How do we want to handle devices without ADM? This includes people who sideload our builds to non-Amazon platforms and whatever we find out above
- If we don't want to handle it, add a Sentry warning so we know how many users are hitting this

Reply:
> If ADM is not available, that means send tab is unavailable on this device

@mcomella @dnarcese FYI, this is not true. When you 'sync' in FxA you can get new tabs from SendTabFeature.

What did before push support was added, was to do a poll `onStart` to check for new tabs, and it would follow the same flow as push if the service is unavailable.

Reply:
Thanks for clarifying, yes I mispoke. That being said, we have historically assumed all devices have push and weren't intending to support polling so we'll need to follow up on what to do for these users. It sounds like adding support is minimal effort so maybe that's what we should do.

Now that I think about it, adding polling support enables us to test on emulators too, which could be valuable.

This can be a follow-up.